src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -1355,7 +1355,7 @@ protected internal override void OnProjectRemoved(ProjectId projectId)
                             _projectSystemNameToProjectsMap.Remove(projectName);
                         }
 
-                        return;
+                        break;
```

Comment:
any chance this loop could have been copied/pasted into other methods in VisualStudioWorkspaceImpl.cs?  Worth a quick audit to look for other potential areas where this might bite us?

Reply:
@CyrusNajmabadi Nope. This was a freshly written mistake just before the holidays.